### ISSUE TITLE
New feature request: Auto-close CE when the attached process is closed or is no longer detected

### ISSUE THREAD

**Issue by wikid24** [cheat-engine/cheat-engine#2909]
Before I start, I just want to say thanks for making a great tool, I've used this app for years now!

Scenario:
- You're playing a single-player game offline and have CE running in the background.
- A friend messages you and wants to play a competitive multiplayer game
- You close the single-player game (forgetting to close CE) and open the multiplayer game
- The multiplayer game has anti-cheat protection and detects CE is running (even though it is not attached to any process)
- Instantly banned on your multiplayer game account 

It's happened before to me a long time ago. I want to prevent this scenario from occurring again. Can I request a new feature? 

Requesting a new checkbox in the CE settings menu that will automatically close CE if the attached process closes or is no longer detected. 

Basically, I want to prevent this scenario from occurring when you forget to close CE and may run the risk of your account being banned. 

Thank you!

**Comment by DeHby** [comment]
I have encountered this problem. I was in PUBG because CE was still running in the background but I didn't notice it. When I logged in again, my account was banned.

But I think the most reliable way is to compile CE yourself (bypass anti-cheat)  For example: [https://www.unknowncheats.me/forum/anti-cheat-bypass/504191-undetected-cheat-engine-driver-2023-bypass-anticheats-eac.html](url)
